test/e2e/util.go

```diff
@@ -17,18 +17,20 @@ limitations under the License.
 package e2e
 
 import (
+	"bytes"
 	"fmt"
-	"math/rand"
-	"path/filepath"
-	"strconv"
-	"time"
-
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/api"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/client"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/client/clientcmd"
 	"github.com/GoogleCloudPlatform/kubernetes/pkg/clientauth"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	"math/rand"
```

Comment:
Please put these back at the top of the import block. See https://github.com/golang/go/wiki/CodeReviewComments#imports